### PR TITLE
Improved _config.yaml and added new style features

### PR DESCRIPTION
**Pull request by lukalafaye**
* Added a few social networks in the config
* Added tables, quotations styles
* Updated links, font-size for headings and text
* Added code highlighting

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -15,7 +15,13 @@ name:
 profile_img: 
 profile: 
 social:
-  github: 
+#  github: https://github.com/username/
+#  twitter: https://twitter.com/username/
+#  youtube: https://www.youtube.com/channel/channelid
+#  codepen: https://codepen.io/username/
+#  linkedin: https://www.linkedin.com/in/username/
+#  https://www.facebook.com/username/
```
Comment (thomasvaeth) [inline _config.yml]: Could you add `facebook: ` before this one?

File: _layouts/post.html [hunk 2]
```diff
@@ -27,9 +27,12 @@ <h1 itemprop="name headline">{{ page.title }}</h1>
...
 
       {{ content }}
-      
+      {% if site.disqus.shortname %}
+        {% include disqus_comments.html %}
+      {% endif %}
     </div>
-  </section>
+    
+  </section
```
Comment (thomasvaeth) [inline _layouts/post.html]: Looks like `section` tag isn't closed here.

**Comment by thomasvaeth** [comment]
I just had those two comments. Once those are updated I will merge this in.

**Comment by lukalafaye** [comment]
Any updates?

**Comment by lukalafaye** [comment]
I added more commits in this PR : 
+ Code highlighting
+ Markdown tables style
+ Improved links style
+ Improved font-size in all posts (removed firstchild) and made everything easier to read!

File: _sass/_base.scss [hunk 3]
```diff
@@ -33,15 +34,15 @@ body {
   line-height: 1.6;
 
   @media (min-width: $mq-xs) {
-    font-size: 102.5%;
+    font-size: 133.25%;
```
Comment (thomasvaeth) [inline _sass/_base.scss:36]: Why are font sizes changed this drastically? I'm not going to merge this in.

File: _sass/_base.scss [hunk 4]
```diff
@@ -11,6 +11,7 @@ $color-alpha--light: #f5f3ef;
 $color-black: #0a0a0a;
 $color-grey: #f8f9f9;
 $color-white: #fff;
+$color-blue: #323CFF;
```
Comment (thomasvaeth) [inline _sass/_base.scss]: Why did you add a new color?

File: _sass/_base.scss [hunk 5]
```diff
@@ -124,27 +137,6 @@ img {
   max-width: 100%;
 }
 
-blockquote {
```
Comment (thomasvaeth) [inline _sass/_base.scss:127]: Why delete the styling for a blockquote?

File: _includes/footer.html [hunk 6]
```diff
@@ -1,3 +1,4 @@
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/rellax/1.0.0/rellax.min.js"></script>
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
 <script type="text/javascript" src="{{ '/assets/js/app.js' | prepend: site.baseurl }}"></script>
+<script id="dsq-count-scr" src="//hackdown.disqus.com/count.js" async></script>
```
Comment (thomasvaeth) [inline _includes/footer.html]: There is an if statement above checking for comments, so shouldn't there be an if statement checking if this script should even be loaded?

File: _includes/disqus_comments.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
+{% if page.comments != false %}
```
Comment (thomasvaeth) [inline _includes/disqus_comments.html]: Where does `page.comments` come from? Shouldn't this be checking if there is a disqus shortname?

**Review by thomasvaeth — CHANGES_REQUESTED** [review]
I don't agree with some of these changes that are now occurring. I left a bunch of comments.

File: _includes/disqus_comments.html (hunk 7 above)
Comment (lukalafaye) [reply _includes/disqus_comments.html]: I'll fix all this and let you know, thanks for the feeback!

File: _config.yml (hunk 1 above)
Comment (lukalafaye) [reply _config.yml]: Done.

File: _layouts/post.html (hunk 2 above)
Comment (lukalafaye) [reply _layouts/post.html]: Closed now.

File: _sass/_base.scss (hunk 3 above)
Comment (lukalafaye) [reply _sass/_base.scss:36]: In my opinion, headings look nicer on my website (see template post). I'm not much experienced with scss/css, would changing the font-size in % in the following style change headings size significantly?

```css
  @media (min-width: $mq-xs) {
    font-size: 133.25%;
  }

  @media (min-width: $mq-sm) {
    font-size: 136%;
  }

  @media (min-width: $mq-md) {
    font-size: 140%;
  }

  @media (min-width: $mq-lg) {
    font-size: 110%;
  }

  @media (min-width: $mq-xl) {
    font-size: 115%;
  }

  @media (min-width: $mq-xxl) {
    font-size: 125%;
  }
}
```
If the answer is yes, does that mean that you think the headings are two big on the template post? Anyway, I can remove them if you'd like to merge the PR, let me know!

File: _sass/_base.scss (hunk 4 above)
Comment (lukalafaye) [reply _sass/_base.scss]: I changed the style for links:

```css
a {
  color: $color-black;
  text-decoration: none;
  transition: color 0.4s ease-in-out;

  &:hover,
  &:active,
  &:focus {
    color: $color-blue;
  }
}
```

If you think the previous style was better, I can reverse this. Let me know what you think!

File: _includes/footer.html (hunk 6 above)
Comment (lukalafaye) [reply _includes/footer.html]: I'll fix all this and let you know, thanks for the feeback!

File: _sass/_base.scss (hunk 5 above)
Comment (lukalafaye) [reply _sass/_base.scss:127]: I wrote a new one (see template post to see how it renders blockquotes):

```css
  blockquote {
      background-color: #eee;
      padding: 0px 52px;
      margin: 8px 0;
      border-radius: 4px;
      position: relative;
      overflow: hidden;
      font-size: 18px;

      &::before,
      &::after {
          content: '“';
          color: #000;
          font-size: 120px;
          position: absolute;
          line-height: 1;
          opacity: 0.2;
      }
      &::before {
          top: -16px;
          left: -4px;
      }

      &::after {
          bottom: -16px;
          right: -4px;
          transform-origin: center;
          transform: rotate(180deg);
      }
  }
```

File: _sass/_base.scss (hunk 4 above)
Comment (lukalafaye) [reply _sass/_base.scss]: The links are now on color-alpha by default and blue on hover, if you'd like me to change the stylr back to how it was before please add a new comment.

File: _includes/disqus_comments.html (hunk 7 above)
Comment (lukalafaye) [reply _includes/disqus_comments.html]: I finally decided to let my friend finish the Disqus comments, I removed them from the forked repo so that you can merge. He'll open a new pr for disqus.

File: _includes/footer.html (hunk 6 above)
Comment (lukalafaye) [reply _includes/footer.html]: Removed disqus comments, consider this fixed.

File: _layouts/post.html [hunk 8]
```diff
@@ -23,14 +23,6 @@ <h1 itemprop="name headline">{{ page.title }}</h1>
     </div>
   </header>
 
-  <section class="section-padding bg-grey" itemprop="articleBody">
```
Comment (thomasvaeth) [inline _layouts/post.html]: Did you run a build? This is the code for the blog post to display, but it has been removed so this needs to be reverted.

Comment (lukalafaye) [reply _layouts/post.html]: Changes were reverted.

File: _sass/_base.scss (hunk 3 above)
Comment (lukalafaye) [reply _sass/_base.scss:36]: I think this is the last change request, can I resolve the conversation? Also if you merge this, it'd be great to have an acknowledgment section in the README.md to thank everyone who made a useful PR (consider the proposition).

**Comment by lukalafaye** [comment]
All change requests have been resolved.
If you merge this, it'd be great to have an acknowledgment section in the README.md to thank everyone who made a useful PR (consider the proposition). Let me know if there is anything else needed!

**Comment by thomasvaeth** [comment]
Thank you for contributing, but unfortunately I am going to have to close without merging. It started off just by adding Disqus comments and cleaning the _config.yml (which was great), but did a little too much I didn't agree with. Some problems were:

- Disqus comments were added/removed
- Blockquote styling was removed
- CSS for code syntax was added twice
- Font size was changed where it is 30% bigger on mobile/tablet
- Styling for something called `.chooser` was added
